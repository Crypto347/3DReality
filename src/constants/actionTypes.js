export const START_TIMER = 'START_TIMER';
export const CHECK_BUTTONS = 'CHECK_BUTTONS';

export const MAKE_PAPERCLIP = 'MAKE_PAPERCLIP';
export const UPDATE_CLIPS_PER_SEC = 'UPDATE_CLIPS_PER_SEC';


export const START_SELLING = 'START_SELLING';
export const UPDATE_FUNDS = 'UPDATE_FUNDS';
export const UPDATE_UNSOLD_INVENTORY = 'UPDATE_UNSOLD_INVENTORY';
export const START_UPDATING_UNSOLD_INVENTORY = 'START_UPDATING_UNSOLD_INVENTORY';
export const STOP_UPDATING_UNSOLD_INVENTORY = 'STOP_UPDATING_UNSOLD_INVENTORY';
export const CALC_DELAY_UNSOLD_INVENTARY = 'CALC_DELAY_UNSOLD_INVENTARY';

export const LOWER_PRICE = 'LOWER_PRICE';
export const RAISE_PRICE = 'RAISE_PRICE';

export const UPDATE_PUBLIC_DEMAND = 'UPDATE_PUBLIC_DEMAND';
export const MARKETING = 'MARKETING';
export const TOGGLE_MARKETING_BUTTON = 'TOGGLE_MARKETING_BUTTON';
export const MARKETING_NEXT_LEVEL = 'MARKETING_NEXT_LEVEL';
export const UPDATE_MAX_PUBLIC_DEMAND = 'UPDATE_MAX_PUBLIC_DEMAND';

export const START_BUYING_WIRE = 'START_BUYING_WIRE';
export const BUY_WIRE = 'BUY_WIRE';
export const RANDOM_WIRE_PRICE = 'RANDOM_WIRE_PRICE';
export const WIRE_PRICE = 'WIRE_PRICE';
export const TOGGLE_WIRE_BUTTON = 'TOGGLE_WIRE_BUTTON';
export const CHECK_EXISTENCE_OF_WIRE = 'CHECK_EXISTENCE_OF_WIRE';
export const STOP = 'STOP';
export const WIRE_EXISTS = 'WIRE_EXISTS';


export const CLICK_WIRE_BUTTON = 'CLICK_WIRE_BUTTON';
export const SHOW_AUTO_WIRE_BUYER = 'SHOW_AUTO_WIRE_BUYER';


export const AUTO_PAPERCLIPS = 'AUTO_PAPERCLIPS';
export const AUTO_CLIPPERS_ADD_ONE = 'AUTO_CLIPPERS_ADD_ONE';
export const AUTO_PAPERCLIPS_START = 'AUTO_PAPERCLIPS_START';
export const TOGGLE_MAKE_PAPERCLIP_BUTTON = 'TOGGLE_MAKE_PAPERCLIP_BUTTON';

export const SET_AUTO_CLIPPER_INIT_PRICE = 'SET_AUTO_CLIPPER_INIT_PRICE';
export const TOGGLE_AUTO_CLIPPERS_BUTTON = 'TOGGLE_AUTO_CLIPPERS_BUTTON';
export const TOGGLE_MEGA_CLIPPERS_BUTTON = 'TOGGLE_MEGA_CLIPPERS_BUTTON';


export const TRUST_PLUS_ONE = 'TRUST_PLUS_ONE';
export const TRUST_PLUS_ONE_FROM_PROJECT = 'TRUST_PLUS_ONE_FROM_PROJECT';
export const INCREASE_OPS = 'INCREASE_OPS';
export const START_DECREASING_OPERATIONS = 'START_DECREASING_OPERATIONS';
export const START_DECREASING_OPS = 'START_DECREASING_OPS';
export const STOP_DECREASING_OPERATIONS = 'STOP_DECREASING_OPERATIONS';
export const DECREASE_OPS = 'DECREASE_OPS';
// export const ADD_CAPTURE_VAL = 'ADD_CAPTURE_VAL';
export const INCREASE_PROCESSORS = 'INCREASE_PROCESSORS';
export const INCREASE_PROCESSORS_MEMORY = 'INCREASE_PROCESSORS_MEMORY';
export const SEND_COMMENT_TO_TERMINAL = 'SEND_COMMENT_TO_TERMINAL';


export const INIT_PROJECTS = 'INIT_PROJECTS';
export const CHECK_CARD_VALIDITY = 'CHECK_CARD_VALIDITY';

export const ADD_PROJECT = 'ADD_PROJECT';
export const ADD_CHIP = 'ADD_CHIP';
export const REMOVE_PRICE_OF_PROJECT_OPS = 'REMOVE_PRICE_OF_PROJECT_OPS';
export const REMOVE_PRICE_OF_PROJECT_CREAT = 'REMOVE_PRICE_OF_PROJECT_CREAT';
export const REMOVE_PRICE_OF_PROJECT_OPS_AND_CREAT = 'REMOVE_PRICE_OF_PROJECT_OPS_AND_CREAT';
export const REMOVE_PRICE_OF_PROJECT_TRUST = 'REMOVE_PRICE_OF_PROJECT_TRUST';
export const DELETE_CARD = 'DELETE_CARD';
export const ADD_NEW_STRATEGY = 'ADD_NEW_STRATEGY';


export const SHOW_REV_TRACKER = 'SHOW_REV_TRACKER';
export const START_CREATIVITY_COUNTER = 'START_CREATIVITY_COUNTER';
export const INCREASE_CREATIVITY = 'INCREASE_CREATIVITY';
export const CREATIVITY_TURN_ON = 'CREATIVITY_TURN_ON';
export const ADD_LEXICAL_PROCESSING = 'ADD_LEXICAL_PROCESSING';
export const ADD_COMBINATORY_HARMONICS = 'ADD_COMBINATORY_HARMONICS';
export const ADD_THE_HADWINGER_PROBLEM = 'ADD_THE_HADWINGER_PROBLEM';
export const ADD_THE_TOTH_SAUSAGE_CONJECTURE = 'ADD_THE_TOTH_SAUSAGE_CONJECTURE';
export const ADD_DONKEY_SPACE = 'ADD_DONKEY_SPACE';
export const SHOW_INVESTMENT_ENGINE = 'SHOW_INVESTMENT_ENGINE';
export const SHOW_STRATEGIC_MODELING = 'SHOW_STRATEGIC_MODELING';
export const TOGGLE_DROPDOWN_INVESTMENTS = 'TOGGLE_DROPDOWN_INVESTMENTS';
export const TOGGLE_DROPDOWN_STRATEGIC_MODELING = 'TOGGLE_DROPDOWN_STRATEGIC_MODELING';
export const CLOSE_DROPDOWNS = 'CLOSE_DROPDOWNS';
export const CHOOSE_FROM_DROPDOWN = 'CHOOSE_FROM_DROPDOWN';
export const ADD_CHOSEN_FROM_DROPDOWN = 'ADD_CHOSEN_FROM_DROPDOWN';



export const SHOW_QUANTUM_COMPUTING = 'SHOW_QUANTUM_COMPUTING';
export const SHOW_QUANT_COMP_MESSAGE = 'SHOW_QUANT_COMP_MESSAGE';
export const TOGGLE_CHIP = 'TOGGLE_CHIP';
export const TOGGLE_WIRE_BUYER_PROJECT = 'TOGGLE_WIRE_BUYER_PROJECT';
export const AUTO_WIRE_BUYER = 'AUTO_WIRE_BUYER';
export const TOGGLE_AUTO_WIRE_BUYER = 'TOGGLE_AUTO_WIRE_BUYER';
export const ADD_MEGA_CLIPPERS = 'ADD_MEGA_CLIPPERS';
export const SHOW_MEGA_CLIPPERS = 'SHOW_MEGA_CLIPPERS';
export const SHOW_AUTO_CLIPPERS = 'SHOW_AUTO_CLIPPERS';




export const CHANGE_TO_Q_OPS = 'CHANGE_TO_Q_OPS';
export const START_ADDING_Q_OPS = 'START_ADDING_Q_OPS';
export const ADD_Q_OPS = 'ADD_Q_OPS';
export const STOP_ADDING_Q_OPS = 'STOP_ADDING_Q_OPS';
export const START_SUBTRACTING_Q_OPS = 'START_SUBTRACTING_Q_OPS';
export const SUBTRACT_Q_OPS = 'SUBTRACT_Q_OPS';
export const STOP_SUBTRACTING_Q_OPS = 'STOP_SUBTRACTING_Q_OPS';
export const CAPTURE_CURRENT_Q_OPS = 'CAPTURE_CURRENT_Q_OPS';
export const UPDATE_OPS = 'UPDATE_OPS';



export const IMPROVE_AUTO_PAPER_CLIPPER = 'IMPROVE_AUTO_PAPER_CLIPPER';
export const IMPROVE_WIRE_EXTRUSION = 'IMPROVE_WIRE_EXTRUSION';
export const START_IMPROVING_MARKETING = 'START_IMPROVING_MARKETING';
export const IMPROVE_MARKETING = 'IMPROVE_MARKETING';

export const START_INVESTMENTS_DEPOSIT = 'START_INVESTMENTS_DEPOSIT';
export const GET_DEPOSIT = 'GET_DEPOSIT';
export const START_UPDATING_SCREEN = 'START_UPDATING_SCREEN';
export const STOP_UPDATING_SCREEN = 'STOP_UPDATING_SCREEN';
export const ADD_INVESTMENTS_LINE = 'ADD_INVESTMENTS_LINE';
export const START_ADDING_EMPTY_INVESTMENTS_LINE = 'START_ADDING_EMPTY_INVESTMENTS_LINE';
export const STOP_ADDING_EMPTY_INVESTMENTS_LINE = 'STOP_ADDING_EMPTY_INVESTMENTS_LINE';
export const START_UPDATING_INVESTMENTS_LINE = 'START_UPDATING_INVESTMENTS_LINE';
export const STOP_UPDATING_INVESTMENTS_LINE = 'STOP_UPDATING_INVESTMENTS_LINE';
export const UPDATE_INVESTMENTS_LINES = 'UPDATE_INVESTMENTS_LINES';
export const START_COUNTING_RISK = 'START_COUNTING_RISK';
export const UPDATE_INVESTMENTS_TOTAL = 'UPDATE_INVESTMENTS_TOTAL';
export const UPDATE_INVESTMENTS_CASH = 'UPDATE_INVESTMENTS_CASH';
export const UPDATE_INVESTMENTS_STOCKS = 'UPDATE_INVESTMENTS_STOCKS';
export const START_APPLYING_PROFIT_LOSS = 'START_APPLYING_PROFIT_LOSS';
export const UPDATE_FAKE_INVESTMENTS_CASH = 'UPDATE_FAKE_INVESTMENTS_CASH';
export const START_INVESTMENTS_WITHDRAW = 'START_INVESTMENTS_WITHDRAW';
export const UPDATE_FUNDS_WITHDRAW = 'UPDATE_FUNDS_WITHDRAW';
export const UPDATE_INVESTMENTS_DELAY = 'UPDATE_INVESTMENTS_DELAY';


export const UPDATE_AVG_REV_PER_SEC = 'UPDATE_AVG_REV_PER_SEC';
export const UPDATE_AVG_CLIPS_SOLD_PER_SEC = 'UPDATE_AVG_CLIPS_SOLD_PER_SEC';

export const MEGA_CLIPPERS_BUTTON_PRESSED = 'MEGA_CLIPPERS_BUTTON_PRESSED';
export const START_MEGACLIPPERS = 'START_MEGACLIPPERS';
export const IMPROVE_MEGA_CLIPPERS = 'IMPROVE_MEGA_CLIPPERS';
export const SWITCH_OFF_OR_ON_AUTO_AND_MEGA_CLIPPERS = 'SWITCH_OFF_OR_ON_AUTO_AND_MEGA_CLIPPERS';

export const START_NEW_TOURNAMENT = 'START_NEW_TOURNAMENT';
export const TOURNAMENT_STATE = 'TOURNAMENT_STATE';
export const UPDATE_NEW_TOURNAMENT_COST = 'UPDATE_NEW_TOURNAMENT_COST';
export const TOGGLE_NEW_TOURNAMENT_BUTTON = 'TOGGLE_NEW_TOURNAMENT_BUTTON';
export const START_RUNNING_STRATEGIC_MODELING = 'START_RUNNING_STRATEGIC_MODELING';
export const UPDATE_STRATEGIC_MODELING_DATA = 'UPDATE_STRATEGIC_MODELING_DATA';
export const UPDATE_STRATEGIC_MODELING_CURRENT_LIST = 'UPDATE_STRATEGIC_MODELING_CURRENT_LIST';
export const STRATEGY_CHOSEN = 'STRATEGY_CHOSEN';
export const CLEAR_CHOSEN_FROM_STRATEGIC_MODELING_DROPDOWN_LIST = 'CLEAR_CHOSEN_FROM_STRATEGIC_MODELING_DROPDOWN_LIST';
export const UPDATE_STRATEGIC_MODELING_ROUND = 'UPDATE_STRATEGIC_MODELING_ROUND';
export const TOURNAMENT_STARTED = 'TOURNAMENT_STARTED';
export const STOP_TOURNAMENT = 'STOP_TOURNAMENT';
export const TOURNAMENT_DURATION = 'TOURNAMENT_DURATION';
// export const UPDATE_NUMBER_OF_ROUND = 'UPDATE_NUMBER_OF_ROUND';
export const SHOW_ROUND_AND_PLAYERS = 'SHOW_ROUND_AND_PLAYERS';
export const START_UPDATING_ROUNDS_ON_SCREEN = 'START_UPDATING_ROUNDS_ON_SCREEN';
export const UPDATE_ROUNDS_ON_SCREEN = 'UPDATE_ROUNDS_ON_SCREEN';
export const START_UPDATING_PLAYER_LEFT_ON_SCREEN = 'START_UPDATING_PLAYER_LEFT_ON_SCREEN';
export const UPDATE_PLAYER_LEFT_ON_SCREEN = 'UPDATE_PLAYER_LEFT_ON_SCREEN';
export const START_UPDATING_PLAYER_TOP_ON_SCREEN = 'START_UPDATING_PLAYER_TOP_ON_SCREEN';
export const UPDATE_PLAYER_TOP_ON_SCREEN = 'UPDATE_PLAYER_TOP_ON_SCREEN';
export const SET_PLAYERS_ARRAY = 'SET_PLAYERS_ARRAY';
export const GAME_STARTED = 'GAME_STARTED';
export const ALL_ROUNDS_RESULT = 'ALL_ROUNDS_RESULT';
export const STRATEGIC_MODELING_RESULT = 'STRATEGIC_MODELING_RESULT';
export const START_COUNTING_RESULT = 'START_COUNTING_RESULT';
export const UPDATED_ALL_ROUNDS_RES = 'UPDATED_ALL_ROUNDS_RES';
export const COUNT_FINAL_RESULT_OF_EACH_STRATEGY = 'COUNT_FINAL_RESULT_OF_EACH_STRATEGY';
export const UPDATE_LIST_OF_FINAL_RESULT = 'UPDATE_LIST_OF_FINAL_RESULT';
export const FILL_WITH_VALUES_STRATEGIC_MODELING_CURRENT_LIST = 'FILL_WITH_VALUES_STRATEGIC_MODELING_CURRENT_LIST';
export const TOGGLE_STRATEGIC_MODELING_LEFT_PART = 'TOGGLE_STRATEGIC_MODELING_LEFT_PART';
export const TOGGLE_STRATEGIC_MODELING_HOVER = 'TOGGLE_STRATEGIC_MODELING_HOVER';
export const TOGGLE_CELLS = 'TOGGLE_CELLS';
export const UPDATE_YOMI = 'UPDATE_YOMI';
