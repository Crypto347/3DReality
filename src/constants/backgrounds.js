export const AnniversaryLounge = 'AnniversaryLounge';
export const AutumnHockey = 'AutumnHockey';
export const BethnalGreenEntrance = 'BethnalGreenEntrance';
export const DresdenMoat = 'DresdenMoat';
export const GrayPier = 'GrayPier';
export const Lebombo = 'Lebombo';
export const MistyPines = 'MistyPines';
export const MusicHall = 'MusicHall';
export const SkukuzaGolf = 'SkukuzaGolf';
export const SnowyPark = 'SnowyPark';
export const SpruitSunrise = 'SpruitSunrise';
export const SunnyVondelpark = 'SunnyVondelpark';
export const UmhlangaSunrise = 'UmhlangaSunrise';
export const UrbanStreet = 'UrbanStreet';
