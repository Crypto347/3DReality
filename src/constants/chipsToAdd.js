export const Chip1 = {
    chipsNumber: 'chip1',
    showChip: false,
    id: "chip1",
    qOps: -360
}

export const Chip2 = {
    chipsNumber: 'chip2',
    showChip: false,
    id: "chip2",
    qOps: -360
}

export const Chip3 = {
    chipsNumber: 'chip3',
    showChip: false,
    id: "chip3",
    qOps: -360
}

export const Chip4 = {
    chipsNumber: 'chip4',
    showChip: false,
    id: "chip4",
    qOps: -360
}

export const Chip5 = {
    chipsNumber: 'chip5',
    showChip: false,
    id: "chip5",
    qOps: -360
}

export const Chip6 = {
    chipsNumber: 'chip6',
    showChip: false,
    id: "chip6",
    qOps: -360
}

export const Chip7 = {
    chipsNumber: 'chip7',
    showChip: false,
    id: "chip7",
    qOps: -360
}

export const Chip8 = {
    chipsNumber: 'chip8',
    showChip: false,
    id: "chip8",
    qOps: -360
}

export const Chip9 = {
    chipsNumber: 'chip9',
    showChip: false,
    id: "chip9",
    qOps: -360
}

export const Chip10 = {
    chipsNumber: 'chip10',
    showChip: false,
    id: "chip10",
    qOps: -360
}
