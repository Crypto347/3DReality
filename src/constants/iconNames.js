export const HOME = 'home';
export const SERVICES = 'services';
export const ABOUT = 'about';
export const GALLERY = 'gallery';
export const CONTACT = 'contact';