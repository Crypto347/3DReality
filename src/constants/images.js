export const images = [
    {
        id: 1,
        src: "pic1",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 2,
        src: "pic2",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 3,
        src: "pic3",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 4,
        src: "pic4",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 5,
        src: "pic5",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 6,
        src: "pic6",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 7,
        src: "pic7",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 8,
        src: "pic8",
        title: "Title",
        description: "Developer",
        hover: false
    },
    {
        id: 9,
        src: "pic9",
        title: "Title",
        description: "Developer",
        hover: false
    }
]
