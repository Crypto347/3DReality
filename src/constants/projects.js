export const AutoClippers50 =   {
    text1: "Even Better AutoClippers (2,500 ops)",
    text2 : "Increases AutoClipper performance by an",
    text3 : "additional 50%",
    price: {
        ops: 3
    },
    action: 50,
    next: 'autoClippers50',
    valid: false,
    id: "card4",
    terminal: "AutoClippper performance boosted by another 50%"
}

export const OptimizedAutoClippers75 =   {
    text1: "Optimized AutoCLippers (5,000 ops)",
    text2 : "Increases AutoClipper performance by an",
    text3 : "additional 75%",
    price: {
        ops: 3
    },
    action: 50,
    next: 'autoClippers75',
    valid: false,
    id: "card5",
    terminal: "AutoClippper performance boosted by another 75%"
}








export const WireExtrusion75 =   {
    text1: "Optimized Wire Extrusion (3,500 ops)",
    text2 : "75% more wire supply from every spool",
    price: {
        ops: 3
    },
    action: 50,
    next: 'wireExtrusion50',
    valid: false,
    id: "card6",
    terminal: "Wire extrusion technique optimized, 2,625 supply from every spool|"
}