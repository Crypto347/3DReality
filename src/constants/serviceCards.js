export const serviceCards = [
    {
        id: 1,
        cardId: "card1",
        icon: "faLaptopCode",
        header: "Header 1",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    },
    {
        id: 2,
        cardId: "card2",
        icon: "faChartLine",
        header: "Header 2",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    },
    {
        id: 3,
        cardId: "card3",
        icon: "faPaperPlane",
        header: "Header 3",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    },
    {
        id: 4,
        cardId: "card4",
        icon: "faUniversity",
        header: "Header 4",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    },
    {
        id: 5,
        cardId: "card5",
        icon: "faCameraRetro",
        header: "Header 5",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    },
    {
        id: 6,
        cardId: "card6",
        icon: "faShoppingBag",
        header: "Header 6",
        text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore.",
        show: false
    }
]
