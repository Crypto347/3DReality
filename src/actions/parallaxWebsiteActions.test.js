/**
* Libraries
*/

import configureStore from 'redux-mock-store';

/**
* Constants
*/

import * as Actions from '.';
import * as actionTypes from "../constants/actionTypes";

/**
* Tests
*/

const mockStore = configureStore();
const store = mockStore();

describe('parallaxWebsiteActions', () => {
    beforeEach(() => { // Runs before each test in the suite
      store.clearActions();
    });

    it('Dispatches the correct action and payload (toggleMenuButton)', () => {
        const expectedActions = [
            {
                type: actionTypes.TOGGLE_MENU_BUTTON
            }
        ];

        store.dispatch(Actions.toggleMenuButton());
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });

    it('Dispatches the correct action and payload (menuButtonIsToggled)', () => {
        const expectedActions = [
            {
                type: actionTypes.MENU_BUTTON_IS_TOGGLED,
                val: true
            }
        ];

        store.dispatch(Actions.menuButtonIsToggled(true));
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });

    it('Dispatches the correct action and payload (initServices)', () => {
        const expectedActions = [
            {
                type: actionTypes.INIT_SERVICES,
                array: [{a: 1, b: 2},{a: 1, b: 2}]
            }
        ];

        store.dispatch(Actions.initServices([{a: 1, b: 2},{a: 1, b: 2}]));
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });

    it('Dispatches the correct action and payload (showCard)', () => {
        const expectedActions = [
            {
                type: actionTypes.SHOW_CARD,
                val: true
            }
        ];

        store.dispatch(Actions.showCard(true));
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });
   
    it('Dispatches the correct action and payload (initTeamMembers)', () => {
        const expectedActions = [
            {
                type: actionTypes.INIT_TEAM_MEMBERS,
                array: [{a: 1, b: 2},{a: 1, b: 2}]
            }
        ];

        store.dispatch(Actions.initTeamMembers([{a: 1, b: 2},{a: 1, b: 2}]));
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });

    it('Dispatches the correct action and payload (initImages)', () => {
        const expectedActions = [
            {
                type: actionTypes.INIT_IMAGES,
                array: [{a: 1, b: 2},{a: 1, b: 2}]
            }
        ];

        store.dispatch(Actions.initImages([{a: 1, b: 2},{a: 1, b: 2}]));
        expect(store.getActions()).toEqual(expectedActions);
        expect(store.getActions()).toMatchSnapshot();
    });
});
