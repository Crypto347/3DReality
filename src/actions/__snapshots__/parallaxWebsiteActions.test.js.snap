// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parallaxWebsiteActions Dispatches the correct action and payload (initImages) 1`] = `
Array [
  Object {
    "array": Array [
      Object {
        "a": 1,
        "b": 2,
      },
      Object {
        "a": 1,
        "b": 2,
      },
    ],
    "type": "INIT_IMAGES",
  },
]
`;

exports[`parallaxWebsiteActions Dispatches the correct action and payload (initServices) 1`] = `
Array [
  Object {
    "array": Array [
      Object {
        "a": 1,
        "b": 2,
      },
      Object {
        "a": 1,
        "b": 2,
      },
    ],
    "type": "INIT_SERVICES",
  },
]
`;

exports[`parallaxWebsiteActions Dispatches the correct action and payload (initTeamMembers) 1`] = `
Array [
  Object {
    "array": Array [
      Object {
        "a": 1,
        "b": 2,
      },
      Object {
        "a": 1,
        "b": 2,
      },
    ],
    "type": "INIT_TEAM_MEMBERS",
  },
]
`;

exports[`parallaxWebsiteActions Dispatches the correct action and payload (menuButtonIsToggled) 1`] = `
Array [
  Object {
    "type": "MENU_BUTTON_IS_TOGGLED",
    "val": true,
  },
]
`;

exports[`parallaxWebsiteActions Dispatches the correct action and payload (showCard) 1`] = `
Array [
  Object {
    "type": "SHOW_CARD",
    "val": true,
  },
]
`;

exports[`parallaxWebsiteActions Dispatches the correct action and payload (toggleMenuButton) 1`] = `
Array [
  Object {
    "type": "TOGGLE_MENU_BUTTON",
  },
]
`;
